### ISSUE TITLE
Bug: Annotator does not work with PDF's generated from PowerPoint 2013

### ISSUE THREAD

**Issue by Saenil** [elias-sundqvist/obsidian-annotator#294]
Hi, I really like the idea and while I was testing it with files from my uni it turned out that it is impossible to annotate anything on them, because everything is rendered as svg (I suppose) and not as text. The typical workflow for generating pdf's at my uni is to create a powerpoint presentation and then export it to pdf format - or at least that is what my profs do.

I know that pdf's can be tricky to work with but maybe there is something that can be done to fix it (or maybe a workaround)?

One interesting thing is that, when I create a new PDF file using PowerPoint from Office 365 then it work correctly, I'm able to create an annotation - even thou it is a bit misaligned in markdown view (but that's probably not related to this issue).

To sum it up:
- it doesn't work with PDF exported from PowerPoint 2013
- it does work with PDF exported from PowerPoint 365
